source/base/Event.ooc

```diff
@@ -17,14 +17,20 @@
 
 Event: class {
 	_head: Func
-	head ::= this _head
 	_tail: This
+	head ::= this _head
```

Comment:
I don't agree. Fields should always grouped at the top. I we decide otherwise, it should be done very, very soon. Since there are *a lot* of files to adapt. @simonmika, do chime in.